frameworks/hdfs/tests/test_sanity.py

```diff
@@ -14,70 +11,62 @@
 import sdk_tasks
 import sdk_upgrade
 import sdk_utils
-from tests.config import (
-    DEFAULT_HDFS_TIMEOUT,
-    DEFAULT_TASK_COUNT,
-    FOLDERED_SERVICE_NAME,
-    PACKAGE_NAME,
-    ZK_SERVICE_PATH,
-    check_healthy,
-    expect_recovery,
-    get_pod_type_instances
-)
+import shakedown
+from tests import config
```

Comment:
Is there an obvious advantage that I'm failing to see over importing this way as opposed to what we're currently doing?